internal/fs/rename_dir_test.go

```diff
@@ -83,6 +84,29 @@ func (t *RenameDirTests) TestRenameFolderWithEmptySourceDirectory() {
 	assert.Equal(t.T(), 0, len(dirEntries))
 }
 
+func (t *RenameDirTests) TestRenameSymlinkToExplicitDir() {
+	targetDirName := "target_dir"
+	err := os.Mkdir(path.Join(mntDir, targetDirName), dirPerms)
+	require.NoError(t.T(), err)
+	oldPath := path.Join(mntDir, "symlink_old")
+	err = os.Symlink(targetDirName, oldPath)
+	require.NoError(t.T(), err)
+	newPath := path.Join(mntDir, "symlink_new")
+
+	err = os.Rename(oldPath, newPath)
+
+	assert.NoError(t.T(), err)
```

Comment:
nit:.
```suggestion
	require.NoError(t.T(), err)
```
at other places too